docs/standard/serialization/system-text-json/customize-properties.md

```diff
@@ -243,35 +243,35 @@ By default, properties are serialized in the order in which they're defined in t
 
 :::code language="csharp" source="snippets/how-to-6-0/csharp/PropertyOrder.cs":::
 
-## Use GitHub Copilot to customize property names and order
+##  Use GitHub Copilot to customize how property names are serialized
 
-You can use GitHub Copilot in your IDE to generate code to customize names and order of serialized properties. You can customize the prompt to output a JSON string with property names and values that suit your requirements.
+You can use GitHub Copilot with `System.Text.Json` attributes to apply patterns of changes to how your code serializes without having to explicitly specify the change in each one.
 
-The following example shows you how to use Copilot to modify existing code to customize property names and order when serializing to JSON.
+Suppose your class declaration has properties that follow `PascalCasing`, and the JSON standard for your project is `snake_casing`. You can use AI to add the necessary `JsonPropertyName` attributes to every property in your class. You can use Copilot to make these changes with a chat prompt like this:
 
-1. Add the following C# example code to a code file `Example.cs` in your editor.
-   In Visual Studio, you can use a C# console application project to try this example.
-
-   :::code language="csharp" source="snippets/how-to-6-0/csharp/copilot-example.cs":::
-
-   `Example.cs` code does the following:
-
-   - Creates an instance of the `Person` class and initializes its properties with values.
-   - Serializes the `person` object to a JSON string using `JsonSerializer.Serialize`.
-   - Prints the following JSON string to the console:
-
-   ```json
-   {"FirstName":"John","LastName":"Doe","Age":30,"Country":"USA"}
-   ```
+```copilot-prompt
+Update #ClassName:
+when the property name contains more than one word,
+change the serialized property name to use underscores between words.
```

Comment:
Can we just ask it to convert to snake casing instead of having to spell it out?

Reply:
Will need to test out any prompt changes. To get this out quickly, ok to do this in a different iteration?